NEWS.md

```diff
@@ -1,3 +1,16 @@
+v0.22 (2016-06-30)
+------------------
+
+- Better unicode handling in commands.
+- When parsing reaction and compound entities from the YAML files, produce
+  better error messages when IDs are invalid.
+- Work around a bug in Cplex that in rare causes a segmentation fault when a
+  linear programming problem is solved repeatedly.
+- Add `fastgapfill` module which allows API access to run fastgapfill
+    algorithm.
+- Add `randomsparse` module which allows API access to generate a random
+    minimum model which satisfies the flux threshold of the objective reaction.
```

Comment:
Maybe prefix these two with API? "API: Add `fastgapfill` module which allow access to run the fastGapFill algorithm."